src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -398,7 +310,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      */
     fetchFeedItems(shouldRefreshCache: boolean = false, shouldDestroy: boolean = false) {
         const { file, api, features } = this.props;
-        const shouldShowNewTasks = isFeatureEnabled(features, 'activityFeed.tasks.newApi');
+        const shouldShowNewTasks = true;
```

Comment:
Can we just get rid of this?

Reply:
It's a positional arg, so until feed api is updated, it needs to be there because it's not the last flag